docs/reference/commandline/volume_rm.md

```diff
@@ -19,6 +19,7 @@ Aliases:
   rm, remove
 
 Options:
+  -f, --force   Force the removal of one or more volumes
```

Comment:
Thanks. Didn't notice the misalignment with the following line `--help` before. Just updated the pull request.